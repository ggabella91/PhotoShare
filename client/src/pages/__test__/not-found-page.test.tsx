import { shallow } from 'enzyme';
import React from 'react';

import '../not-found/not-found-page.component';
import NotFoundPage from '../not-found/not-found-page.component';

it('renders a not-found page component', () => {
  const notFoundPageWrapper = shallow(<NotFoundPage />);

  expect(notFoundPageWrapper).toMatchSnapshot();
});
