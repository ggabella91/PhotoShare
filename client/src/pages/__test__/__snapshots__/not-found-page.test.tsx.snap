// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a not-found page component 1`] = `
<div
  className="not-found-page"
>
  <h2
    className="not-found-header"
  >
    Sorry, this page isn't available.
  </h2>
  <div
    className="not-found-body"
  >
    <span
      className="not-found-body-text"
    >
      The link you followed may be broken, or the page may have been removed.
      <NavLink
        className="link-text"
        to="/me"
      >
         
        Go back to PhotoShare.
      </NavLink>
    </span>
  </div>
</div>
`;
